modules/psap-installing-node-feature-discovery-operator.adoc

```diff
@@ -0,0 +1,144 @@
+
+As a cluster administrator, you can install the Operator using the CLI.
+
+.Prerequisites
+
+* An {product-title} cluster
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a namespace for the Node Feature Discovery Operator.
+
+.. Create the following `Namespace` custom resource (CR) that defines the `openshift-nfd-operator` namespace, and then save the YAML in the `nfd-namespace.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-nfd-operator
+----
+
+.. Create the namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create -f nfd-namespace.yaml
+----
+
+. Install the NFD Operator in the namespace you created in the previous step by creating the following objects:
+
+.. Create the following `OperatorGroup` CR and save the YAML in the `nfd-operatorgroup.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  generateName: openshift-nfd-
+  name: openshift-nfd
+  namespace: openshift-nfd-operator
+spec:
+  targetNamespaces:
+  - openshift-nfd-operator
+
+----
+
+.. Create the `OperatorGroup` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f nfd-operatorgroup.yaml
+----
+
+.. Run the following command to get the `channel` value required for the next step.
++
+[source,terminal]
+----
+$ oc get packagemanifest openshift-nfd-operator -n openshift-marketplace -o jsonpath='{.status.defaultChannel}'
+----
++
+.Example output
+[source,terminal]
+----
+nfd.v4.8.0
+----
+
+.. Create the following `Subscription` CR and save the YAML in the `nfd-sub.yaml` file:
++
+.Example Subscription
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: nfd
+  namespace: openshift-nfd-operator
+spec:
+  channel: nfd.v4.8.0
+  installPlanApproval: Automatic
+  name: nfd
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+----
+
+.. Create the subscription object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f nfd-sub.yaml
+----
+
+.. Change to the `openshift-nfd-operator` project:
++
+[source,terminal]
+----
+$ oc project openshift-nfd-operator
+----
+
+[id="install-operator-web-console_{context}"]
+== Installing the NFD Operator using the web console
+
+As a cluster administrator, you can install the Node Feature Discovery Operator using the web console.
+
+[NOTE]
+====
+You must create the `Namespace` CR and `OperatorGroup` CR as mentioned in the previous section.
+====
+
+.Procedure
+
+. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+. Choose *Node Feature Discovery* from the list of available Operators, and then click *Install*.
+
+. On the *Install Operator* page, select *A specific namespace on the cluster*, select the namespace reacted in the previous section, and then click *Install*.
```

Comment:
"A specific namespace on the cluster" - lowercase a
"select the namespace reacted in the previous section" - created instead of reacted.